sql-parser/dialect/oracle/src/main/antlr4/imports/oracle/Number.g4

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+lexer grammar Number;
```

Comment:
Why add new Number.g4?

Reply:
I think it's because numbers don't fit in existing files like `Alphabet`.

Reply:
@strongduanmu Cuz the keyword " REGR_R2" has number, I cannot add it to the keyword.g4 directly. I have discussed it with @RaigorJiang in the morning, and the solution we find out is to add a number.g4 file to solve it.

Reply:
Thank you for your explanation